config/routes.rb

```diff
@@ -27,6 +27,8 @@
                  constraints: ->(_) { FeatureFlipper.show_education_benefit_form? }
     end
 
+    resources :health_care_application, only: [:create, :index]
```

Comment:
For right now they can only have 1 application since there is no authentication, each application is anonymous.